app-frontend/src/app/pages/login/login.controller.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+import assetLogo from '../../../assets/images/logo-raster-foundry.png';
+import ffcSpace from '../../../assets/images/ffc-space.png';
+import videom4v from '../../../assets/video/ffc-space.m4v';
+import videoogg from '../../../assets/video/ffc-space.ogg';
+import videowebm from '../../../assets/video/ffc-space.webm';
+
+class MainController {
+    constructor($log) {
+        'ngInject';
+        $log.debug('Lazy Loaded  login controller initialized');
```

Comment:
Should probably remove the references to lazy loading since that isn't happening any longer.